dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsApiExtendedServiceImpl.java

```diff
@@ -290,4 +274,41 @@ private void deleteIndex(RestClient restClient, String index) {
             LOG.warn("Could not delete previous elastic search " + index + " index, not an issue if this is cold start", e);
         }
     }
+
+    private Response searchIndex(String query, MultivaluedMap<String, String> queryParameters, String index) {
+        if (!config.getEsConfiguration().getHostname().isEmpty()) {
+            try (RestClient restClient = RestClient
+                    .builder(new HttpHost(config.getEsConfiguration().getHostname(), config.getEsConfiguration().getPort(), "http"))
+                    .build()) {
+                HttpEntity entity = query == null ? null : new NStringEntity(query, ContentType.APPLICATION_JSON);
+                Map<String, String> parameters = new HashMap<>();
+                // TODO: note that this is lossy if there are repeated parameters
+                // but it looks like the elastic search http client classes don't handle it
+                if (queryParameters != null) {
+                    queryParameters.forEach((key, value) -> parameters.put(key, value.get(0)));
+                }
+                org.elasticsearch.client.Response get = restClient.performRequest("GET", "/" + index + "/_search", parameters, entity);
+                if (get.getStatusLine().getStatusCode() != HttpStatus.SC_OK) {
+                    throw new CustomWebApplicationException("Could not submit " + index + "index to elastic search",
```

Comment:
I don't know ES terminology very well, but it seems like you're querying an index, not submitting one.